api/generator.py

```diff
@@ -0,0 +1,77 @@
+import face_recognition
+import os
+import cv2
+
+
+KNOWN_FACES_DIR = 'api/known_faces'
+TOLERANCE = 0.6
+FRAME_THICKNESS = 3
+FONT_THICKNESS = 1
+MODEL = 'cnn'
+
+known_faces = []
+known_names = []
+
+for name in os.listdir(KNOWN_FACES_DIR):
+    try:
+        for filename in os.listdir(f'{KNOWN_FACES_DIR}/{name}'):
```

Comment:
You may use `os.path.join` for building file paths in more cross-platform manner.

Reply:
> You may use `os.path.join` for building file paths in more cross-platform manner.

Ok, will do it.